backend/wayland/output.c

```diff
@@ -369,5 +366,14 @@ struct wlr_output *wlr_wl_output_create(struct wlr_backend *wlr_backend) {
 
 void wlr_wl_output_set_title(struct wlr_output *output, const char *title) {
 	struct wlr_wl_output *wl_output = get_wl_output_from_output(output);
+
+	if (title == NULL) {
+		char wl_title[32];
```

Comment:
Can you move this declaration right before the `if`? I've seen cases where the compiler will decide to replace the local variable with other stuff outside of the block.

Reply:
Sure, force pushed the change.